moveit_ros/trajectory_cache/include/moveit/trajectory_cache/utils/utils.hpp

```diff
@@ -56,6 +58,21 @@ std::string getWorkspaceFrameId(const moveit::planning_interface::MoveGroupInter
 std::string getCartesianPathRequestFrameId(const moveit::planning_interface::MoveGroupInterface& move_group,
                                            const moveit_msgs::srv::GetCartesianPath::Request& path_request);
 
+/** @brief Restates a translation and rotation in a new frame.
+ *
+ * @param[in] tf. The transform buffer to use.
+ * @param[in] target_frame. The frame to restate in.
+ * @param[in] source_frame. The frame to restate from.
+ * @param[in,out] translation. The translation to restate. Ignored if nullptr.
+ * @param[in,out] rotation. The rotation to restate. Ignored if nullptr.
+ * @returns MoveItErrorCode::SUCCESS if successfully restated. Otherwise, will return return
+ * MoveItErrorCode::FRAME_TRANSFORM_FAILURE if the transform could not be retrieved.
+ */
+moveit::core::MoveItErrorCode restateInNewFrame(std::shared_ptr<tf2_ros::Buffer> tf, const std::string target_frame,
+                                                const std::string source_frame, geometry_msgs::msg::Point* translation,
```

Comment:
I think the 2 strings should be const ref

Reply:
Oops f123763eb91e485f33c1210032299eb6c22551c9